tests/snapshots/tests.integrations.laminas.v1_4.common_scenarios_test.test_scenario_get_to_missing_route.json

```diff
@@ -0,0 +1,310 @@
+[[
+  {
+    "name": "laminas.request",
+    "service": "test_laminas_20",
+    "resource": "GET /does_not_exist",
+    "trace_id": 0,
+    "span_id": 1,
+    "parent_id": 1625545689840073902,
```

Comment:
Why is this parent_id hardcoded? I wonder, are we incidentally doing distributed tracing from the testrunner or such?

Reply:
I think that's because in our traces, we are setting a `parent_id` on the root span (even if that's the same as the `trace_id` in the root span). Therefore, on the [test-agent side](https://github.com/DataDog/dd-apm-test-agent/blob/793ec3c107ffcb420d924ad3ff9f1397f7ed106d/ddapm_test_agent/trace.py#L146), since it sees a `parent_id`, it uses this one.

Below is an example of a root span for the same query:
```
{
            "trace_id": "2161630347673372554",
            "span_id": "7241179534834786024",
            "name": "laminas.request",
            "resource": "Application\\Controller\\CommonSpecsController@simple simple",
            "service": "test_laminas_20",
            "start": 1685602874633867000,
            "error": 0,
            "type": "web",
            "duration": 339176000,
            "parent_id": "2161630347673372554",
            "meta": {
                "_dd.p.dm": "-1",
                "http.url": "http:\/\/localhost:9999\/simple?key=value&<redacted>",
                "http.method": "GET",
                "span.kind": "server",
                "component": "laminas",
                "http.version": "1.1",
                "http.route": "simple",
                "laminas.route.action": "Application\\Controller\\CommonSpecsController@simple",
                "http.status_code": "200"
            },
            "metrics": {
                "process_id": 12829,
                "_sampling_priority_v1": 1,
                "php.compilation.total_time_ms": 79.091
            }
        }
```

Reply:
Other question: does the parent id disappear, if we add `DD_DISTRIBUTED_TRACING=0` to the phpunit runner?

Reply:
Yep, running `DD_DISTRIBUTED_TRACING=0 DD_TRACE_CLI_ENABLED=true php  -d ddtrace.request_init_hook=/home/circleci/app/bridge/dd_wrap_autoloader.php ./tests/vendor/bin/phpunit  --config=./tests/phpunit.xml tests/Integrations/Laminas/V2_0 --filter=.`, I no longer have the `parent_id` in the root span (and `metrics._dd.rule_psr` now appears, just noting it).

Reply:
This would be sort of the expected output, and we need to change things so that phpunit itself will be called with `-d datadog.distributed_tracing=0`. But that's not for this PR.